docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -14,6 +14,25 @@ additional driver and software components are properly installed and running.
 
 [Intel SGX]: https://www.intel.com/content/www/us/en/architecture-and-technology/software-guard-extensions.html
 
+## BIOS Configuration
+To set up a TEE (Intel SGX), you need to configure BIOS. First, **update BIOS 
+to the latest version with the latest microcode**. Then proceed with BIOS 
```

Comment:
Even better:
```suggestion
to the latest version of the microcode** and then proceed with BIOS 
```

Reply:
This would actually not be correct as the BIOS version is separate from the microcode version and the BIOS _contains_ the signed microcode update.